documentation/docs/20-core-concepts/10-routing.md

```diff
@@ -205,6 +205,8 @@ We can create a layout that only applies to pages below `/settings` (while inher
 <slot></slot>
 ```
 
+Note that if you want to leverage an `#each` block to dynamically build the nav menu in the code above, you will need to define the _sections_ array.  See `+layout.js` below for the matching example.
```

Comment:
This could be made much shorter. I'd prefer to avoid starting it with "Note" as it sounds like some sort of gotcha when it's really just the basic straightforward functioning of the system. How about this?
```suggestion
You can see how `data` is populated in the `+layout.js` example in the next section just below.
```